lib/Dialect/FIRRTL/FIRRTLOps.cpp

```diff
@@ -283,16 +283,22 @@ void FModuleOp::build(OpBuilder &builder, OperationState &result,
 
   if (annotations)
     result.addAttribute("annotations", annotations);
+  if (result.attributes.get("argNames"))
+    return;
+  SmallVector<Attribute> argNames;
+  for (size_t i = 0, e = ports.size(); i != e; i++)
+    argNames.push_back(builder.getStringAttr(""));
+  result.addAttribute("argNames", builder.getArrayAttr(argNames));
```

Comment:
I think we're guaranteed to have argNames and this whole block of code can be removed?

Reply:
buildModule just created argNames.